WolvenKit/WolvenKit.csproj

```diff
@@ -35,6 +35,8 @@
 
   <ItemGroup>
     <PackageReference Include="CommunityToolkit.Mvvm" Version="8.0.0" />
+    <PackageReference Include="Nodify" Version="1.7.0" />
```

Comment:
Hi, I'm happy to see that you use Nodify! 
You seem to be using an older version which is missing lots of new features and fixes which you can read about in the [changelog](https://github.com/miroiu/nodify/blob/master/CHANGELOG.md).